Gemfile.lock

```diff
@@ -0,0 +1,1581 @@
+  remote: https://github.com/samvera-labs/hyrax-iiif_av.git
+  revision: 4723b8fd0961fa179caf787532bd7922a44957ec
+  branch: rails_hyrax_upgrade
+  specs:
+    hyrax-iiif_av (0.2.0)
+      blacklight
+      hyrax (>= 3.5, < 6.0)
+      iiif_manifest (> 0.5)
+      rails (>= 5.1, < 8.0)
+
+GIT
+  remote: https://github.com/samvera/bulkrax.git
+  revision: d8f9b85a9064b352a252c5deb590a430ca4a738e
+  branch: main
+  specs:
+    bulkrax (8.0.0)
+      bagit (~> 0.4.6)
+      coderay
+      denormalize_fields
+      iso8601 (~> 0.9.0)
+      kaminari
+      language_list (~> 1.2, >= 1.2.1)
+      libxml-ruby (~> 3.2.4)
+      loofah (>= 2.2.3)
+      marcel
+      oai (>= 0.4, < 2.x)
+      rack (>= 2.0.6)
+      rails (>= 5.1.6)
+      rdf (>= 2.0.2, < 4.0)
+      rubyzip
+      simple_form
+
+GIT
+  remote: https://github.com/samvera/hyrax.git
+  revision: 74f29943d88749a57077818d271b29729a6d7183
+  branch: double_combo
+  specs:
+    hyrax (5.0.1)
+      active-fedora (~> 14.0)
+      almond-rails (~> 0.1)
+      awesome_nested_set (~> 3.1)
+      blacklight (~> 7.29)
+      blacklight-gallery (~> 4.0)
+      breadcrumbs_on_rails (~> 3.0)
+      browse-everything (>= 0.16, < 2.0)
+      carrierwave (~> 1.0)
+      clipboard-rails (~> 1.5)
+      concurrent-ruby (~> 1.0)
+      connection_pool (~> 2.4)
+      draper (~> 4.0)
+      dry-container (~> 0.11)
+      dry-events (~> 1.0, >= 1.0.1)
+      dry-logic (~> 1.5)
+      dry-monads (~> 1.6)
+      dry-validation (~> 1.10)
+      flipflop (~> 2.3)
+      flot-rails (~> 0.0.6)
+      font-awesome-rails (~> 4.2)
+      google-analytics-data (~> 0.6)
+      hydra-derivatives (~> 3.3)
+      hydra-editor (~> 6.0)
+      hydra-file_characterization (~> 1.1)
+      hydra-head (~> 12.0)
+      hydra-works (>= 0.16)
+      iiif_manifest (>= 0.3, < 2.0)
+      json-schema
+      legato (~> 0.3)
+      linkeddata
+      mailboxer (~> 0.12)
+      nest (~> 3.1)
+      noid-rails (~> 3.0)
+      oauth
+      oauth2 (~> 1.2)
+      openseadragon
+      posix-spawn
+      qa (~> 5.5, >= 5.5.1)
+      rails (~> 6.1)
+      rails_autolink (~> 1.1)
+      rdf-rdfxml
+      rdf-vocab (~> 3.0)
+      redis (~> 4.0)
+      redis-namespace (~> 1.5)
+      redlock (>= 0.1.2, < 2.0)
+      reform (~> 2.3)
+      reform-rails (~> 0.2.0)
+      retriable (>= 2.9, < 4.0)
+      sass-rails (~> 6.0)
+      select2-rails (~> 3.5)
+      signet
+      sprockets (= 3.7.2)
+      tinymce-rails (~> 5.10)
+      valkyrie (~> 3.1.1)
+      view_component (~> 2.74.1)
+
+GIT
+  remote: https://github.com/scientist-softserv/apartment.git
+  revision: 9ad4b0ef2ee6debb956e86ef9e7095c8358fef5e
+  branch: development
+  specs:
+    apartment (2.2.1)
+      activerecord (>= 3.1.2, < 7.0)
+      parallel (>= 0.7.1)
+      public_suffix (>= 2)
+      rack (>= 1.3.6)
+
+GIT
+  remote: https://github.com/scientist-softserv/iiif_print.git
+  revision: 805191e6caedf2b5b91eb4e0aad692d4aff871c1
+  branch: main
+  specs:
+    iiif_print (2.0.0)
```

Comment:
Shouldn't we update to 2.0.1 to bring in the bug fixes?

Reply:
Yep. I'll update hyku and redo this then.

Reply:
@laritakr https://github.com/samvera/hyku/pull/2243